frontend/src/descriptions/submitBookKeeping.jsx

```diff
@@ -0,0 +1,37 @@
+export const submitBookKeeping =
+    "15 läsdagar innan varje sektionsmöte (oftast torsdag LP3) måste sektionens kassörer skicka ett mail till [revisorerna](mailto:revisorer@chalmers.it) som uppfyller följande instruktioner:  \n" +
+
+    "**Ämnesrad vid ordinarie granskning:**  \n" +
+    "`<KOMMITÉNAMN> Sektionsmöte LP<LP> bokföring`  \n" +
+    "**Ämnesrad vid granskning inför ansvarsbefrielse:**  \n" +
+    "`<KOMMITÉNAMN+MANDATÅR> ansvarsbefrielse LP<LP>`  \n" +
+    "Ex: Säg att FooIT18/19 vill lämna in sin bokföring inför sektionsmötet LP1, då skickar kassören ett mail med ämnesraden  \n" +
+    "`FooIT Sektionsmöte LP1 bokföring`  \n" +
+    "Ex: FooIT18/19 vill i LP2 lämna in sin bokföring för att bli ansvarsbefriade, då skickar kassören ett mail med ämnesraden  \n" +
+    "`FooIT18/19 ansvarsbefrielse LP2`  \n" +
+
+    "##### I mailet ska det finnas följande dokument med tillhörande filnamn:\n" +
+    "- **Verifikationslista** (hittas i FortNox): `<KOMMITÉNAMN+VERKSAMHETSÅR>_lp<LP>_verifikat.pdf`\n" +
+    "- **Huvudbok** (hittas i FortNox): `<KOMMITÉNAMN+VERKSAMHETSÅR>_lp<LP>_huvudbok.pdf`\n" +
+    "- **Resultatrapport** (hittas i FortNox): `<KOMMITÉNAMN+VERKSAMHETSÅR>_lp<LP>_resultat.pdf`\n" +
+    "- **Balansrapport** (hittas i FortNox): `<KOMMITÉNAMN+VERKSAMHETSÅR>_lp<LP>_balans.pdf`\n" +
+    "- **Kontoplan** (hittas i FortNox): `<KOMMITÉNAMN+VERKSAMHETSÅR>_lp<LP>_konto.pdf`\n" +
```

Comment:
Is this really a thing? I think I've never handed it in :thinking:

Reply:
It is defined as necessary under "kvartalsrapport" in [Kassör 101](https://styrit.chalmers.it/wp-content/uploads/Kassor-101.pdf). It is the document used the least, but it can still be useful e.g. if a committé has a different set of accounts than normal.